src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/ReimbursementRequestDetailsView.tsx

```diff
@@ -150,6 +170,16 @@ const ReimbursementRequestDetailsView: React.FC<ReimbursementRequestDetailsViewP
       ]}
       headerRight={<ActionsMenu buttons={buttons} />}
     >
+      <NERModal
```

Comment:
Id make this a component prob in the same file but yk the DeleteModal = () => {... and then call <DeleteModal/>